install.md

```diff
@@ -232,12 +230,30 @@ To configure the installation settings:
 
 ### <a id="opt-out-telemetry"></a>Opting out of telemetry reporting
 
-To disable telemetry reporting in your Application Service Adapter installation:
-
-1. Set the value of the `telemetry.enabled` property in the `tas-adapter-values.yml` file to `false`.
-
-After you install Application Service Adapter via the steps below, it will not send telemetry data to VMware.
-
+By default, when you install Tanzu Application Service Adapter, you are opted into telemetry collection. To turn off telemetry collection, complete following the instructions.
+
+1. Ensure your Kubernetes context is pointing to the cluster where Tanzu Application Service Adapter is installed.
+
+2. Run the following kubectl command:
```

Comment:
I think this is fine for now. Do we know if the `tanzu` CLI's telemetry plugin is ready yet for doing this?

Reply:
TAP ships with Tanzu cli version 0.11.2 and it doesn't seem to have the telemetry plugin
```
$tanzu plugin list
  NAME                DESCRIPTION                                                                   SCOPE       DISCOVERY  VERSION  STATUS
  login               Login to the platform                                                         Standalone  default    v0.11.2  installed
  management-cluster  Kubernetes management-cluster operations                                      Standalone  default    v0.11.2  installed
  package             Tanzu package management                                                      Standalone  default    v0.11.2  installed
  pinniped-auth       Pinniped authentication operations (usually not directly invoked)             Standalone  default    v0.11.2  installed
  secret              Tanzu secret management                                                       Standalone  default    v0.11.2  installed
  accelerator         Manage accelerators in a Kubernetes cluster                                   Standalone             v1.1.1   installed
  apps                Applications on Kubernetes                                                    Standalone             v0.5.1   installed
  insight             post & query image, package, source, and vulnerability data                   Standalone             v1.1.2   installed
  services            Discover Service Types, Service Instances and manage Resource Claims (ALPHA)  Standalone             v0.2.0   installed
```

Reply:
Maybe we can just link to the TAP docs at https://docs.vmware.com/en/Tanzu-Application-Platform/1.1/tap/GUID-opting-out-telemetry.html#turn-off-telemetry-collection-0?

Reply:
We thought about doing that initially. Is that ok though since the documentation linked there specifically mentions just TAP telemetry and not TAS Adapter telemetry? Would that be confusing for users being redirected there?